components/admin_console/admin_definition.jsx

```diff
@@ -1070,6 +1070,10 @@ const AdminDefinition = {
                 'admin.cluster.UseIpAddressDesc',
                 'admin.cluster.UseExperimentalGossip',
                 'admin.cluster.UseExperimentalGossipDesc',
+                'admin.cluster.EnableExperimentalGossipEncryption',
+                'admin.cluster.EnableExperimentalGossipEncryptionDesc',
+                'admin.cluster.EncryptionKey',
```

Comment:
I think this line and the one below is dead code?

Reply:
Yep.